### PR TITLE
Description was shortened 

### PR DESCRIPTION
**Pull request by mgrinko**

### CODE REVIEW

File: src/deepEqual.js [hunk 1]
```diff
@@ -3,10 +3,8 @@
 /**
  * Implement deepEqual function:
  *
- * Function compares 2 values
- * for primitives it compares types and values of given params
- * for objects it checks all fields and compare primitive values
- * inside all fields. Nested objects are compared too.
+ * Non object types are compared with ===
```
Comment (roman-mate) [inline src/deepEqual.js]: nit: don't put line breaks in random places. Line break within 1 paragrapth should be when we reach 80 char limit. In this case line break should be after own. 

* Non object types are compared with ===. Objects are equal if all the own 
* enumerable properties are equal

Comment (mgrinko) [reply src/deepEqual.js]: Why do you think it is better?
For me it is easier to understand that it is another sentence if I put it on it's own new line
rather then search for the dot in the middle of long line

Comment (roman-mate) [reply src/deepEqual.js]: In writings (documents, emails etc.) we use '.' to separate different sentences. jsdocs aren't different from that perspective. 

example of google documentation: 
https://github.com/google/closure-library/blob/master/closure/goog/collections/sets.js#L28

example of facebook documentation:
https://github.com/facebook/react/blob/master/scripts/jest/matchers/reactTestMatchers.js#L7

They all do line breaks at 80 characters. Not at sentences.

Comment (mgrinko) [reply src/deepEqual.js]: Here is an opposite example from the next file to the file you gave from jest
https://github.com/facebook/react/blob/master/scripts/jest/matchers/toWarnDev.js#L48

Comment (mgrinko) [reply src/deepEqual.js]: And here is an example from google closure repo
https://github.com/google/closure-library/blob/master/closure/goog/color/color.js#L38

Comment (mgrinko) [reply src/deepEqual.js]: So I would not say it is so strict to follow close to 80 length
But I agree that in most cases it males sense

Comment (roman-mate) [reply src/deepEqual.js]: first example is weird. they do ',' and then capital case. More of a example of bad documentation.

the second example is an example of bullet points. Yes, bullet points we do from new lines. 

https://google.github.io/styleguide/jsguide.html#jsdoc-line-wrapping
`but this horizontal alignment is discouraged.`

Comment (mgrinko) [reply src/deepEqual.js]: Like it!
Let's add bullet points here for each of 2 options

**Pull request merged**